### ISSUE TITLE
Bug: Minecraft crashes when loading Optifine 1.15.2pre2 along with WWE 1.15.2

### ISSUE THREAD

**Issue by SpazmasterYT** [THEREALWWEFAN231/wwe#458]
**WWE version:** 
1.15.2 version 3.0.4
**Minecraft version:** 
1.15.2
**Operating system:** 
Windows 10
**Optifine/Other mods:** 
Optifine, Litematica for 1.15.2, running on Fabric 0.4.32 build 292

**Whats the issue:** 
Using Optifabric beta3 to load optifine preview 1.15.2_HD_G1_pre2, it results in the game crashing immediately. Without optifine or optifabric present in the mods folder, the game runs fine. Could be due to both optifine's and optifabric's early development, but it seems to be on wwe's side.  Fixing this now could lead to issues down the road not being a problem.

**Crash-report(Would be very useful also please format):**

Mixin prepare failed preparing MixinOptifineBlockModelRenderer in mixins.wwefan.optifine.json: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:855)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:715)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:648)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:515)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:440)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:422)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:285)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:191)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.loadClassData(KnotClassDelegate.java:174)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:143)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
Caused by: java.lang.ClassNotFoundException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.loadMixinClass(MixinInfo.java:1297)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:848)
	... 20 more

**Comment by THEREALWWEFAN231** [comment]
Its seems to be a wwe issue, ill take a look later, it appears to be my xray fix for 1.14.4 that no longer works.

**Comment by thegreatestminer** [comment]
Also occurs on MultiMC. Mods loaded: `optifabric-1.0.0-beta4.jar, preview_OptiFine_1.15.2_HD_U_G1_pre9.jar, wwe 1.15.2.jar`

```
[13:56:08] [main/WARN]: Reference map 'optifabric-refmap.json' for optifabric.optifine.mixins.json could not be read. If this is a development environment you can ignore this message
[13:56:08] [main/FATAL]: Mixin prepare failed preparing MixinOptifineBlockModelRenderer in mixins.wwefan.optifine.json: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:855) ~[sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:715) ~[sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:648) ~[sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:515) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:440) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:422) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:285) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:191) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.7.8+build.184.jar:0.8+build.18+unknown-b18.git-unknown]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.loadClassData(KnotClassDelegate.java:174) [fabric-loader-0.7.8+build.184.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:143) [fabric-loader-0.7.8+build.184.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) [?:1.8.0_221]
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_221]
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) [?:1.8.0_221]
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048) [?:1.8.0_221]
	at java.lang.Class.getMethod0(Class.java:3018) [?:1.8.0_221]
	at java.lang.Class.getMethod(Class.java:1784) [?:1.8.0_221]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192) [fabric-loader-0.7.8+build.184.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138) [fabric-loader-0.7.8+build.184.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26) [fabric-loader-0.7.8+build.184.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_221]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_221]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_221]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_221]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.ClassNotFoundException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.loadMixinClass(MixinInfo.java:1297) ~[sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:848) ~[sponge-mixin-0.8+build.18.jar:0.8+build.18+unknown-b18.git-unknown]
	... 28 more
Failed to start Minecraft:
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [mixins.wwefan.optifine.json:MixinOptifineBlockModelRenderer] from phase [DEFAULT] in config [mixins.wwefan.optifine.json] FAILED during PREPARE
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:643)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinPrepareError(MixinProcessor.java:591)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:518)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.select(MixinProcessor.java:440)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.checkSelect(MixinProcessor.java:422)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:285)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:191)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.loadClassData(KnotClassDelegate.java:174)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:143)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	... 8 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:855)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepareMixins(MixinConfig.java:715)
	at org.spongepowered.asm.mixin.transformer.MixinConfig.prepare(MixinConfig.java:648)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.prepareConfigs(MixinProcessor.java:515)
	... 25 more
Caused by: java.lang.ClassNotFoundException: The specified mixin 'me.THEREALWWEFAN231.wwefan.mixin.mixins.MixinOptifineBlockModelRenderer' was not found
	at org.spongepowered.asm.mixin.transformer.MixinInfo.loadMixinClass(MixinInfo.java:1297)
	at org.spongepowered.asm.mixin.transformer.MixinInfo.<init>(MixinInfo.java:848)
	... 28 more
Exiting with -1
Process exited with code -1.
```

**Comment by THEREALWWEFAN231** [comment]
I forgot to close this, currently, wwe's Xray doesn't work with optifine but optifine now loads with wwe, "This has been added for donators with prebuilds, it will be released for everyone in 3.1."
